### ISSUE TITLE
request: include depth argument for 'isvisible' message in [pdcontrol]

### ISSUE THREAD

**Issue by porres** [pure-data/pure-data#803]
a depth message can query if parent window is visible or not, in a similar fashion to 'dir/args' messages.

**Comment by millerpuckette** [comment]
Adding features is how software inflates and eventually dies of bloat.  One wants a valid use case before just adding stuff.  And even then, if there's already a way to do the same thing, one should think twice.

**Comment by porres** [comment]
yeah, like I said in the pd-list, I had a use case for it which I didn't need in the end, and now I don't even know of a use case for this as it is :) so if no one really needs this (as I expect anyway), I'll just close this.

ps. would still like to know a use case for this just by sheer curiosity ;)

**Comment by porres** [comment]
Well, like I said in the Pd list, I can see how this is useful now. And then the depth argument would follow the same syntax and idea of the 'dir' and 'args' message, because it's really useful when used inside abstractions. So I'll keep the request and maybe give it a go sometime soon.
